pyanaconda/errors.py

```diff
@@ -205,6 +208,16 @@ def _bootloader_error_handler(self, exn):
         else:
             return ERROR_RAISE
 
+    def _non_critical_error_handler(self, exn):
+        message = _("The following error occurred during the installation:")
+        message += "\n\n" + str(exn) + "\n\n"
+        message = _("Would you like to ignore this and continue with installation?")
```

Comment:
As a translator, I would definitely prefer one string to many small pieces. If it's not inconvenient, please make it one string with format or concatenation, as in the handlers above.

Reply:
Fixed.